experiments/readme.md

```diff
@@ -0,0 +1,22 @@
+# Extracting compression messages from the DreamCoder PLDI artifact
```

Comment:
* link to the artifact https://dl.acm.org/do/10.1145/3410302/full/
* mention the whole `./extract_all_data.sh` thing

Reply:
Added in [4f3276d](https://github.com/mlb2251/stitch/pull/87/commits/4f3276df5f0afd915092b1254ef8a9223ff07acc)